### PR TITLE
[#1290] Prevented up/down keys moving template page when track events se...

### PR DESCRIPTION
**Pull request by mjschranz**
...lected. Prevented spacebar from moving page down

https://webmademovies.lighthouseapp.com/projects/65733/tickets/1290-pressing-updown-arrows-while-a-track-event-is-select-is-causing-pages-to-scroll

### CODE REVIEW

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @jbuck received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/78587a3114ce6f3/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @jbuck received. Current queue size: 1

Live output at: http://botio.butyr.org:8000/d326a423e0e5ce5/output.txt

File: src/ui/ui.js [hunk 1]
```diff
@@ -329,6 +331,11 @@ define( [ "core/eventmanager", "./toggler", "./logo-spinner", "./context-button"
         var track,
             trackEvent,
             nextTrack;
+
+        if( butter.selectedEvents.length ) {
```
Comment (jbuck) [inline src/ui/ui.js]: whitespace `if(`

File: src/ui/ui.js [hunk 2]
```diff
@@ -354,6 +361,11 @@ define( [ "core/eventmanager", "./toggler", "./logo-spinner", "./context-button"
...
```
Comment (jbuck) [inline src/ui/ui.js]: same as above

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/78587a3114ce6f3/output.txt

Total script time: 0.67 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/d326a423e0e5ce5/output.txt

Total script time: 1.20 mins

Server is running on http://pr353.butyr.org/templates/test.html

**Pull request merged**